store/views.py

```diff
@@ -63,7 +83,17 @@ def loanBookView(request):
     '''
     # START YOUR CODE HERE
     book_id = None # get the book id from post data
-
+    book_id = request.POST['bid']
+    obj=Book.objects.get(id=book_id)
```

Comment:
You are directly accessing POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).

This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.